### ISSUE TITLE
Could you provide the final model for AFEW?

### ISSUE THREAD

**Issue by shadowclouds** [Open-Debin/Emotion-FAN#15]
Hello , Thanks for your excellent work! Could you share the final trained model for AFEW8.0, that we email to the  dhallabhinav@gmail.com and other posters that you mentioned in the issue, but do not get reply.
We only want to use your final model to produce a demo for each video.  And we will give the demo to you if we finish. Could you help us? 
Thanks so much!

**Comment by Open-Debin** [comment]
@shadowclouds 
Merry Christmas! I recently update the Emotion-FAN, new features include data process, environment install, CK+ code, and Baseline code. Also, you can find the old version directory of Emotion-FAN in the README.md. I hope my new updates can help you greatly. Please see the Emotion-FAN for more details.

**Comment by Rasipuram** [comment]
Thank you for sharing updates on Emotion-FAN. 

I am running fan_afew_manifest.py code on AFEW faces generated using the same code. But the model shows training accuracy of 41% and validation accuracy of 7.38% (constant) after 180 epochs. I have not modified any parts of the code and followed all the steps. Can you please help in re-producing the results shared in the paper?
